nautobot/ipam/api/serializers.py

```diff
@@ -481,6 +481,25 @@ def validate(self, data):
         return super().validate(data)
 
 
+class VLANAllocationSerializer(NautobotModelSerializer, TaggedModelSerializerMixin):
+    """
+    Input serializer for POST to /api/ipam/vlan-groups/<id>/available-vlans/, i.e. allocating VLAN from VLANGroup.
+    """
+
+    class Meta(VLANSerializer.Meta):
+        model = VLAN
+        fields = (
+            # not vid, vlan_group as those are implied by the selected VLANGroup
+            "name",
+            "status",
+            "role",
+            "tenant",
+            "locations",
```

Comment:
Do we actually support setting the `locations` M2M via this API? Normally that'd be done as a separate step via the `vlan-location-assignments` endpoint. If not supported here let's remove it.

Reply:
Removed